elitech/__init__.py

```diff
@@ -40,7 +40,7 @@
 class Device:
     def __init__(self, serial_port):
         if serial_port is not None:
-            self._ser = serial.Serial(serial_port, 115200, timeout=3)
+            self._ser = serial.Serial(serial_port, baudrate=115000, timeout=3)
```

Comment:
Not quite the same code - examine the baudrate value carefully.  000 instead of 200.

I came across this solution by Googling "pyserial 115000 115200" when I discovered altering the baudrate slightly corrected the problem.  This seems to be a common problem with people using Arduinos and Raspberry Pis.